net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+					For Net-SNMP 5.0.9, or any Net-SNMP with patch 722075 applied, set this value to 2.<br/>
+					A value of 2 is equivalent to -IR on Net-SNMP command line utilities.<br/>
+					UCD-SNMP and Net-SNMP 5.0.8 and previous may not be able to translate certain formats of<br/>
+					symbolic names such as RFC1213-MIB::sysDescr. Net-SNMP 5.0.9 or patch 722075 will allow<br/>
+					all possibilities to be translated.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Log Trap OID</fielddescr>
+			<fieldname>translate_log_trap_oid</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to use the default of numerical OID<br/>
+					Set to 1 to translate the trap OID to short text (symbolic form) (eg: linkUp)<br/>
+					Set to 2 to translate the trap OID to short text with module name (eg: IF-MIB::linkUp)<br/>
+					Set to 3 to translate the trap OID to long text (eg: iso...snmpTraps.linkUp)<br/>
+					Set to 4 to translate the trap OID to long text with module name (eg:<br/>
+					IF-MIB::iso...snmpTraps.linkUp)<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>2</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Value OIDs</fielddescr>
+			<fieldname>translate_value_oids</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable translating OID values to text (symbolic form)<br/>
+					Set to 1 to translate OID values to short text (symbolic form) (eg: BuildingAlarm)<br/>
+					Set to 2 to translate OID values to short text with module name (eg: UPS-MIB::BuildingAlarm)<br/>
+					Set to 3 to translate OID values to long text (eg: iso...upsAlarm.BuildingAlarm)<br/>
+					Set to 4 to translate OID values to long text with module name (eg:<br/>
+					UPS-MIB::iso...upsAlarm.BuildingAlarm)<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Enterprise OID Format</fielddescr>
+			<fieldname>translate_enterprise_oid_format</fieldname>
+			<description>
+				<![CDATA[
+					Configures how the symbolic enterprise OID will be displayed for $E.<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.